tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -212,3 +212,17 @@ extension XCScheme.ArchiveAction {
         )
     }
 }
+
+extension XCScheme.BuildableReference: Hashable {
```

Comment:
Should this be in XcodeProj instead? We can submit a PR, and use a patch until it's accepted.

Reply:
Potentially, yes. To keep the custom scheme train moving, I would like to do this here for now.

Reply:
I can track it as a TODO in #573.